### DISCUSSION TITLE
x-xss-protection = 0 on Nextcloud AIO

### DISCUSSION THREAD

**Discussion by psychedelicu** [ZoeyVid/NPMplus#801] [Q&A] [answered]
Hi @Zoey2936,

Some headers are already set in the Nextcloud installation /htaccess file which can lead to errors if you set them again in nginx. 
add_header X-XSS-Protection "1; mode=block"; is present in nextcloud .htaccess.

Do you have any ideea why is 0 when using NPM Plus? I have tried to add it again in NPM settings, but then it appears twice, first "0" and the second to "1", but the tests still shows 0.  

Maybe is addeed somewhere in NPM Plus? Because i can't find it. 

![image](https://github.com/ZoeyVid/NPMplus/assets/35727400/33eeb958-58be-4d7c-a67f-12449168c056)

<img width="579" alt="image" src="https://github.com/ZoeyVid/NPMplus/assets/35727400/9bcdc7ea-2e0f-443c-9082-3f8713d02244">

**Comment by Zoey2936** [comment, marked as answer]
see here: https://github.com/ZoeyVid/NPMplus/discussions/501

**Comment by psychedelicu** [comment]
Oh, ok. Great then, doing nothing is the be thing that i can do! Thanks! :D
